docusaurus/docusaurus.config.js

```diff
@@ -24,81 +30,7 @@ const config = {
       "@docusaurus/plugin-client-redirects",
       {
         fromExtensions: ["html", "htm"], // /myPage.html -> /myPage
-        redirects: [
-          // /docs/oldDoc -> /docs/newDoc
-          {
-            from: "/airbyte-pro",
-            to: "/airbyte-enterprise"
-          },
-          {
-            from: "/upgrading-airbyte",
-            to: "/operator-guides/upgrading-airbyte",
-          },
-          {
-            from: "/catalog",
-            to: "/understanding-airbyte/airbyte-protocol",
-          },
-          {
-            from: "/integrations/sources/google-analytics-data-api",
-            to: "/integrations/sources/google-analytics-v4",
```

Comment:
:information_source: This was actually ignored, since both files exist, so I removed it in the yml.